src/Microsoft.Identity.Web/TokenAcquisition.Logger.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Identity.Web
+{
+    /// <summary>
+    /// LoggingMessage class for TokenAcquisition.
+    /// </summary>
+    internal partial class TokenAcquisition
```

Comment:
Not sure if it's worth splitting this class for Logging.

Reply:
this is what the docs recommend doing for this type of logging. i've included the links above in the  PR description.